README.rst

```diff
@@ -22,8 +22,8 @@ The `PyPA recommended`_ tool for installing Python packages.
 * `Issue Tracking`_
 * `User mailing list`_
 * `Dev mailing list`_
-* User IRC: #pypa on Freenode.
-* Dev IRC: #pypa-dev on Freenode.
+* `User IRC`_: #pypa on Freenode
+* `Dev IRC`_: #pypa-dev on Freenode.
```

Comment:
A period is removed, I assume by accident? Also the punctuation is inconsistent in this list; this may be a chance to fix it.

Reply:
Removed periods in bullet points as periods look a bit out of place in bullet points without complete sentences.

Reply:
Removed text after the last two bullet points for consistency across the whole list.